integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -185,6 +207,18 @@ def __init__(self, *args, **kwargs):
         self.fields["deepl_midyear_start_enabled"].initial = bool(
             self.instance.deepl_midyear_start_month
         )
+        self.disabled_offer_options = (
+            [
+                offer_template
+                for offer_template in Page.objects.filter(region=self.instance)
+                .values_list("embedded_offers", flat=True)
+                .distinct()
+                if offer_template
+            ]
+            if self.instance
+            else []
+        )
```

Comment:
This is not working as intended, because apparently, when creating a new region, the form has its instance set as a hidden region(?), and somehow `OfferTemplate.objects.filter(pages__region=self.instance)` then returns 3 `OfferTemplate`s.

I am really not sure how or why

Reply:
I've pushed the current state, if you could take a look at this I'd really appreciate it.

TBF the previous soution did work even if it was inelegant, so worst case we could also revert to it 😅